mbtiles/src/bin/mbtiles.rs

```diff
@@ -104,6 +111,38 @@ enum Commands {
         #[arg(long, value_enum)]
         agg_hash: Option<AggHashType>,
     },
+    /// Pack a directory tree of tiles into an MBTiles file
+    #[command(name = "pack")]
+    Pack {
+        /// directory to read
+        input_directory: PathBuf,
+        /// MBTiles file to write
+        output_file: PathBuf,
+        /// Tile ID scheme for input directory
+        #[arg(long, value_enum, default_value = "xyz")]
+        scheme: TileScheme,
+    },
+    /// Unpack an MBTiles file into a directory tree of tiles
+    #[command(name = "unpack")]
+    Unpack {
+        /// MBTiles file to read
+        input_file: PathBuf,
+        /// directory to write
+        output_directory: PathBuf,
+        /// Tile ID scheme for output directory
+        #[arg(long, value_enum, default_value = "xyz")]
+        scheme: TileScheme,
+    },
+}
+
+#[derive(Clone, Copy, PartialEq, Debug, clap::ValueEnum)]
+enum TileScheme {
+    /// XYZ (aka. "slippy map") scheme where Y=0 is at the top
+    #[value(name = "xyz")]
+    Xyz,
+    /// TMS scheme where Y=0 is at the bottom
+    #[value(name = "tms")]
+    Tms,
```

Comment:
nit: imo, this is a bit more ergonomic, but both are fine.

```suggestion
        /// Tile ID scheme for input directory
        #[arg(long, value_enum, default)]
        scheme: TileScheme,
    },
    /// Unpack an MBTiles file into a directory tree of tiles
    #[command(name = "unpack")]
    Unpack {
        /// MBTiles file to read
        input_file: PathBuf,
        /// directory to write
        output_directory: PathBuf,
        /// Tile ID scheme for output directory
        #[arg(long, value_enum, default)]
        scheme: TileScheme,
    },
}

#[derive(Clone, Copy, Default, PartialEq, Debug, clap::ValueEnum)]
enum TileScheme {
    /// XYZ (aka. "slippy map") scheme where Y=0 is at the top
    #[value(name = "xyz")]
    #[default]
    Xyz,
    /// TMS scheme where Y=0 is at the bottom
    #[value(name = "tms")]
    Tms,
```

Reply:
nice!